blog/migrations/0031_alter_blogpage_body.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.18 on 2023-05-15 21:22
+
+import common.models.helpers
+from django.db import migrations
+import statistics.registry
+import wagtail.blocks
+import wagtail.embeds.blocks
+import wagtail.fields
+import wagtail.images.blocks
+
+
+def migrate_simple_incidents(apps, schema_editor):
+    blog_pages = apps.get_model('blog', 'BlogPage')
+    for blog_page in blog_pages.objects.all():
+        for block in blog_page.body:
+            if block.block_type == 'vertical_bar_chart':
+                incident_set = block.value.get('incident_set')
+                if incident_set.get('category') is not None:
+                    incident_set['categories'] = [incident_set.get('category')]
+                    blog_page.save()
```

Comment:
@chigby took a quick first pass here, it seems like it works locally but some things feel a little strange to me.

1. Do I need to explicitly set `incident_set['categories']` to `[]` if the prior category is `None`?
2. is `blog_page.save()` the right call here?

Reply:
1. I'm realizing that I possibly misspoke (mis-typed?) when I said this earlier, as I think I was imagining the alternative solution of migrating the `category` field from a single value to a list in-place, rather that what we're actually doing here, which is having two fields side-by-side. So sorry about that, and no, I think in this case we don't need to set the categories to `[]` -- it already will be that (if I understand correctly).
2. Yes, `save()` is correct.

Reply:
To be clear, I think this migration works perfectly. It converted the charts I had locally so the correct category was in the new field, which is exactly what it's supposed to, from where I sit.